gammapy/data/event_list.py

```diff
@@ -654,6 +473,232 @@ def plot_time_map(self, ax=None):
         return ax
 
 
+class EventList(EventListBase):
+    """Event list for IACT dataset
+
```

Comment:
Please add a `Parameters` section here, with an entry `table` just like you have in the base class.
That's one thing in Python and many other languages: there's a lot of duplication in constructors / init, if a sub-class needs a new initialiser.

In this case, I think you can completely remove `EventList.__init__`, because it just calls the base class initialiser, and the same call happens if you don't define an `__init__` in the subclass.
The docstring should still document the `table` parameter in both classes.

Generally, for every pull request you should check the HTML docs locally via:
```
make clean
python setup.py build_docs
open docs/_build/html/index.html
```
Unfortunately this is very slow, so I usually run this, then do something else (e.g. coding or writing tests or something else completely) and then check the docs and fix my docstrings).

Reply:
do you want me to do the same for the `EventListLAT`? Copy paste the docstring from the base class?

Reply:
No, not a blind copy. The goal is to have it show useful info in the HTML docs. To avoid copy you could also add a sentence like:
```
For further information, see the base class: `~gammapy.data.EventListBase`.
```